documentation/modules/ROOT/pages/panache.adoc

```diff
@@ -51,11 +65,12 @@ Add the following database properties to your `application.properties` so that i
 # Configuration file
 # key = value
 greeting=Hello y'all!
-quarkus.datasource.jdbc.url=jdbc:h2:mem:default
```

Comment:
I think the Dev Service should be in its own section because, if we just show them with Dev Services we are skipping important teaching stuff and it is how to configure the JDBC connection.

Another option might be to not create a section but add an INFO admonition explaining how to configure the JDBC connection in case of not using Dev Services.

Reply:
Also, can you add a TIP explaining how to disable DevServices?

Reply:
Sure.

Reply:
In my latest commit, I put this back and explained how to disable the Dev Services.